wallet-new/integration/Error.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+-- | Types describing runtime errors related to
+-- wallet integration tests.
+
+module Error
+    ( IntegrationTestError (..)
+    ) where
+
+import qualified Data.Text.Buildable
+import           Formatting (bprint, stext, (%))
+import           Universum
+
+newtype IntegrationTestError =
+    -- | Some internal error.
+    Internal Text
```

Comment:
I would prefer to see constructors rather than `Text` errors. We can easily add constructors when we need them, but factoring out from a `Text` representation is more work.

Reply:
Fair enough, will fix.